core/network/corda-interop-app/build.gradle

```diff
@@ -15,9 +17,9 @@ buildscript {
         corda_platform_version = constants.getProperty("platformVersion").toInteger()
         arrow_version = constants.getProperty("arrowVersion")
         weaver_version = constants.getProperty("weaverVersion")
-        artifactory_username = ""
-        artifactory_password = ""
-        artifactory_url = ""
+        artifactory_username = constants.getProperty("username")
```

Comment:
Can we rename all "artifactory" references in variable names to "github"?

Also rename the `artifactory.properties` and `artifactory.properties.template` files accordingly.

Reply:
Done.